android/build.gradle

```diff
@@ -38,6 +38,10 @@ apply plugin: 'com.android.library'
 android {
     compileSdkVersion 31
     ndkVersion "20.1.5948944"
+    // Condition for namespace compatibility in AGP 8
+    if (project.android.hasProperty("namespace")) {
```

Comment:
Probably android tools need to be bumped to the newer version `'com.android.tools.build:gradle:8.2.2'`
At the moment CI fails